alexmenuet-alexmenuet-taulia/hw4/BookManager/config/routes.rb

```diff
@@ -1,9 +1,15 @@
 Rails.application.routes.draw do
-    get '/' => "books#index"
-    get '/edit/:id' => "books#edit"
-    post '/' => "books#submit"
-    post '/' => "books#save"
-    post '/' => "books#search"
+    resources :stores do
+       get '/library' => "stores#show" 
+       patch '/library/' => "stores#add"
+    end
+    get '/books' => "books#index"
+    get '/books/:id/edit' => "books#edit", :as => :book
+    get '/books/:id' => "books#delete"
+    put '/books/:id/edit' => "books#update"
+    patch '/books/:search' => "books#search"
+    post '/books' => "books#save"
+    
     
```

Comment:
See if you can use `resources` for all of your routes. Check out `except` and `only` parameters. For example, I could say I only want index and show routes. I'd say, `resources :books, only: [:index, :show]`. I think you'll find them much more clean while still highly customizable.